packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationProfile.tsx

```diff
@@ -57,7 +57,7 @@ export const OrganizationProfileModal = (props: OrganizationProfileProps): JSX.E
   };
 
   return (
-    <Route path='organizationProfile'>
+    <Route path='organization'>
```

Comment:
❓ Is this change intentional? Won't this affect our current component routing?

Reply:
@SokratisVidros this corresponds to the `startPath` props passed in VirtualRouter(Components.tsx). This change was made for consistency(we have `sign-in` not `signIn`)

cc: @nikosdouvlis

Reply:
@nikosdouvlis ☝️

Reply:
I think this is safe, but Id rather  not include it in this PR if its not 100% required as the routing logic is pretty complex